seed.js

```diff
@@ -1,29 +1,39 @@
 const db = require('./database/index-db.js').connection;
 const faker = require('faker');
 
-const seedingQuery = 'insert into product_images (product_id, product_name, media_type, image_desc, image_loc) values (?, ?, ?, ?, ?)'
+const seedingQuery = 'insert into product_images (product_id, product_name, media_type, image_desc, image_loc, brand_name, category, image_num) values (?, ?, ?, ?, ?, ?, ?, ?)'
 
 db.query('truncate table PRODUCT_IMAGES');
 
 fakeDatamaker = () => {
   let product_id = 1;
   let productName = faker.commerce.productName();
+  let description = faker.commerce.productDescription();
+  let brand_name = faker.company.companyName();
+  let category = `Target/Pets/Cat Stuff/${faker.commerce.department()}`;
```

Comment:
Minor Detail that does not affects FEC, but can you clarify this category?
Is it supposed to be categories for Cat Stuff, so like cat toys, cat bed, cat clothes?

Reply:
good point. I definitely owe you an explanation here. 
I just wanted to make sure the category is not static so I can see how the layout was rendering every time I call for a new product. 
I will be adding more info to the DB once the overall structure is set.